main/res/values/strings.xml

```diff
@@ -1544,6 +1545,66 @@
     <string name="from_clipboard">From clipboard</string>
     <string name="copy_to_clipboard">Copy to clipboard</string>
 
+    <!-- Variables -->
+    <string name="variables_titletext">Use this page to note down values and do calculations necessary for this cache. Tap \'Help\' for more information.</string>
+    <string name="variables_scanlisting">Scan listing</string>
+    <string name="variables_scanlisting_choosepattern_title">Choose found pattern</string>
+    <string name="variables_tidyup">Tidy Up</string>
+    <string name="variables_deleteall">Delete All</string>
+    <string name="variables_deleteall_confirm_text">Do you really want to delete all variables on this cache?</string>
+    <string name="variables_info">Help</string>
+
+    <!-- Calculated coordinates -->
+    <string name="calccoord_guided_mode">Guided Mode</string>
+    <string name="calccoord_convert_to_plain">Convert to plain Coordinate</string>
+    <string name="calccoord_remove_spaces">Remove Spaces</string>
+
+    <string name="calccoord_generate_title">Generate Waypoints</string>
+    <string name="calccoord_generate_showonmap">Show on Map</string>
+    <string name="calccoord_generate_waypointnameprefix">Generated</string>
+    <string name="calccoord_generate_error_novarwithrange">No valid variable with ranges found</string>
+    <string name="calccoord_generate_error_novalidgeopoints">No combination returned valid geopoints</string>
+    <string name="calccoord_generate_error_nogeopointselected">No Geopoint selected</string>
+
+    <string name="calccoord_migrate_title">Waypoint Migration</string>
+    <string name="calccoord_migrate_migrate">Migrate</string>
+    <string name="calccoord_migrate_migrate_usernote_praefix">Migrated from</string>
+    <string name="calccoord_migrate_cancel">Cancel</string>
+    <string name="calccoord_migrate_dismiss">Dismiss data</string>
+    <string name="calccoord_migrate_dismiss_usernote_praefix">Dismissed</string>
+    <string name="calccoord_migrate_infotext_markdown">Legacy local calculation coordinate data was found in this waypoint:\n\n`%1$s`\n\nIt is suggested to migrate this to global calculation coordinates as follows:\n\n`%2$s`\n\nwith new cache variables:%3$s\n\nPlease choose:\n- **Migrate** to perform migration as suggested\n- **Dismiss Data** to dismiss the found calculation data for good\n- **Cancel** to continue without migration. You will be asked for migration again next time you open the waypoint for editing. Note however that editing the waypoint coordinates could delete local calculation data</string>
```

Comment:
Suggestion (but maybe its also wise to cut this string into pieces, which can be handled easier):

A legacy coordinate calculation was found in this waypoint:\n\n`%1$s`\n\nIt You can migrate this to the new cache calculation coordinates as follows:\n\n`%2$s`\n\nwith new cache variables:%3$s\n\nPlease choose:\n- **Migrate** to perform migration as suggested\n- **Dismiss Data** to dismiss the found calculation data for good\n- **Cancel** to continue without migration. You will be asked for migration again next time you open the waypoint for editing. Note however that editing the waypoint coordinates could delete local calculation data.